graphql_client_codegen/src/codegen.rs

```diff
@@ -140,10 +149,20 @@ pub fn response_for_query(
             } else {
                 None
             }
-        }).collect();
+        })
+        .collect();
 
     let response_derives = context.response_derives();
 
+    let respons_data_struct_name = if multiple_operation {
+        Ident::new(
+            format!("{}ResponseData", operation.name).as_str(),
+            Span::call_site(),
+        )
+    } else {
+        Ident::new("ResponseData", Span::call_site())
+    };
```

Comment:
If all the queries are in same module, we should give unique name to ResponseData and Variables.